### ISSUE TITLE
Upgrade to 3.44.0 issue

### ISSUE THREAD

**Issue by fmarzocca** [jaedb/Iris#489]
**Describe the bug**
After upgrade to python3 version (`sudo pip3 install Mopidy-Iris`):
- in "version" Iris reports "`Unknown installed`". Pressing the button "Upgrade to 3.44.0" I had these errors in syslog: 
`Feb  8 13:36:26 pab mopidy[1137]: INFO     [Thread-60] mopidy_iris.system Running system action 'upgrade'
Feb  8 13:36:26 pab mopidy[1137]: INFO     [Thread-60] mopidy_iris.system cd /iris && git checkout master && git pull origin master
Feb  8 13:36:26 pab mopidy[1137]: /usr/local/lib/python3.7/dist-packages/mopidy_iris/system.sh: line 12: cd: /iris: No such file or directory
Feb  8 13:36:26 pab mopidy[1137]: INFO     [Thread-60] mopidy_iris.system
Feb  8 13:36:26 pab mopidy[1137]: INFO     [Thread-61] mopidy_iris.system Running system action 'restart'
Feb  8 13:36:26 pab mopidy[1137]: INFO     [Thread-61] mopidy_iris.system Cannot restart Mopidy when running in a Docker container
`.

I think the origin of the error comes from system.sh:
```
if [[ "$(pwd)" = "/iris" ]]; then
        IS_CONTAINER=true
else
        IS_CONTAINER=false
fi
```
This code believes I am in a container, but I am not using Docker.

In addition, shouldn't this line in system.sh be changed to pip3??
`UPGRADE="$(sudo pip install --upgrade mopidy-iris)"`

**Environment details:**
 - OS: Raspbian Buster
 - Browser [ chrome, safari]
 - Iris version 3.44.0

**Screenshot:**
The screenshot has been taken from a running v.3.44.0

![Schermata 2020-02-08 alle 13 55 19](https://user-images.githubusercontent.com/8530030/74085600-dc420080-4a7a-11ea-9749-5e0147401be5.png)

**Comment by JayGatsby7** [comment]
I have the same question about unknown installed. Have a good day!  Thanks!

**Comment by fmarzocca** [comment]
Thanks. This will fix the "update not happening" issue. But still remains the "Unknown version installed" thing.

**Comment by jjok** [comment]
I'm seeing the "Upgrade to 3.44.0" after installing that version from the Github release.

I also noticed that when Mopidy starts up it says `Starting Iris 3.0.0`.

**Comment by arpena** [comment]
> I'm seeing the "Upgrade to 3.44.0" after installing that version from the Github release.
> 
> I also noticed that when Mopidy starts up it says `Starting Iris 3.0.0`.

Not in my case. In /var/log/daemon.log I have:
[IrisFrontend-12] mopidy_iris.core Starting Iris 3.44.0

But still the UI says:
Version: Unknown installed (Upgrade Available)

**Comment by jaedb** [comment]
Other issues resolved, except for what is covered in #499
